react-native-external-accessory.podspec

```diff
@@ -0,0 +1,19 @@
+require 'json'
+
+package = JSON.parse(File.read(File.join(__dir__, 'package.json')))
+
+Pod::Spec.new do |s|
+  s.name         = package['name']
+  s.version      = package['version']
+  s.summary      = package['description']
+  s.license      = package['license']
+
+  s.authors      = package['author']
+  s.homepage     = package['homepage']
+  s.platform     = :ios, "9.0"
+
+  s.source       = { :git => "https://github.com/dbolcsfoldi/react-native-external-accessory" }
+  s.source_files  = "ios/**/*.{h,m}"
+
+  s.dependency 'React'
```

Comment:
Seems like the 'React' dependency is causing `pod lib lint` to fail due to 'React' being deprecated. I rely on react link for integrating the library so it'd be good if you could just quickly summarize the intended use of the podspec and if the `pod lib lint` error is something to be concerned about.

Reply:
Apologies for not being more clear for what this PR was for!

With react-native 0.60.0, it now supports autolinking. Which is to say the `react-native link react-native-external-accessory` is now longer required with the adding of the podspec file. [Here's some autolinking info](https://github.com/react-native-community/cli/blob/master/docs/autolinking.md) and the mentioned [react-native-webview podspec example](https://github.com/react-native-community/react-native-webview/blob/master/react-native-webview.podspec).

As for the React deprecation you mentioned I'm not sure 😕